project_task_default_tag/models/project.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2018 PESOL (http://pesol.es)
+#                Angel Moya (angel.moya@pesol.es)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from odoo import fields, models, api
+
+
+class ProjectProject(models.Model):
+    _inherit = 'project.project'
+
+    tag_ids = fields.Many2many(
```

Comment:
IMO this should be named `default_tag_ids`.
The `tag_ids` fields should be used for a feature adding Projects level Tags.